Doc/library/marshal.rst

```diff
@@ -49,17 +49,16 @@ For format *version* lower than 3, recursive lists, sets and dictionaries cannot
 be written (see below).
 
 There are functions that read/write files as well as functions operating on
-strings.
+bytes-like objects.
 
 The module defines these functions:
 
 
 .. function:: dump(value, file[, version])
 
    Write the value on the open file.  The value must be a supported type.  The
-   file must be an open file object such as ``sys.stdout`` or returned by
-   :func:`open` or :func:`os.popen`.  It must be opened in binary mode (``'wb'``
-   or ``'w+b'``).
+   file must be an open file object opened in binary mode (``'wb'`` or
+   ``'w+b'``).
```

Comment:
I think examples of open modes can be omitted. They are more specific for Python 2 (there is even special C API for `FILE*`).